docs/src/components/ReactExample/components/Playground.tsx

```diff
@@ -71,90 +71,86 @@ const Playground = ({ exampleKnobs, onChange }: Props) => {
             </Text>
             <StyledKnobsWrapper>
               {sortBy(knobs, ["type"]).map(({ type, name, options }) => {
-                const fieldName = `${component}-${name}`;
+                const baseKnobProps = {
+                  key: name,
```

Comment:
I don't think `key` can be spread, see the [`jsx-key`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md) rule, they must be used directly. Although I'm a little confused that there apparently aren't any warnings, but maybe React removes these warnings in production.